android/app/build.gradle

```diff
@@ -197,11 +240,13 @@ android {
 
 dependencies {
     addUnimodulesDependencies()
+
```

Comment:
If it's not related to the PR objective, please don't change it.
We can do it in a future refactor PR.

Reply:
Sorry, but is was a small change and since the PR objective touched on it I believe we don't have any problems on doing that! Any improvement, more yet when it is very small, is always welcome on any PR and in a such codebase that can leads us to forget it later.